string label # text label of the detected object
float64 center_x # bounding box center x coordinate normalized to 0, 1
float64 center_y # bounding box center y coordinate normalized to 0, 1
float64 width # bounding box width
float64 height # bounding box height